packages/frontend/src/components/MenuBar/menubar.module.css

```diff
@@ -138,4 +114,110 @@
   display: flex;
   flex-direction: column;
   align-items: flex-end;
+}
+
+.menuList {
+  background: #fff;
+  overflow: visible;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  margin-top: 0.5rem;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  width: 24.375rem;
+  border-radius: 0.5rem;
+  z-index: 1001;
+}
+
+.menuItems {
+  position: absolute;
+  top: 100%;
+  right: 0;
+  width: auto;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  z-index: 1001;
+}
+
+@media screen and (max-width: 768px) {
+  .menuItems {
+    position: absolute;
+    top: 100%;
+    left: 0;
+    width: 100%;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    z-index: 1001;
+    position: fixed;
+    top: 9.375rem;
+    left: 0;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    z-index: 1001;
+  }
+
+  .menuList {
+    width: 100%;
+    min-height: 100vh;
+  }
+}
+
+.menuItem {
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+  justify-content: space-between;
+  flex-direction: column;
+  padding: 0.375rem;
+  min-height: 3.5rem;
+
+  >svg {
+    min-width: 100%;
+    min-height: 100%;
+  }
+
+}
+
+.menuProfileInfo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 0.625rem;
+  justify-content: space-between;
+  padding-left: 0.625rem;
+  margin-bottom: 1rem;
+}
+
+.menuColumn {
+  display: flex;
+  align-items: center;
+  gap: 0.625rem;
+}
+
+.crossSquare {
+  min-width: 2.25rem;
+  min-height: 2.25rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 8px;
+  font-size: 1.25rem;
+  border: 1px solid #fff;
+}
+
+.menuItem2.menuItem2.menuItem2 {
```

Comment:
hva er dette?

Reply:
klassenavn er veldig lite beskrivende, er det snakk om menuItemLevel2? en variant?

Reply:
Var for å øke specificity, men ser nå at jeg ikke trang det. Navnet endrer jeg så klart.